packages/account/src/Components/personal-details/__tests__/personal-details.spec.tsx

```diff
@@ -415,8 +344,8 @@ describe('<PersonalDetails/>', () => {
 
         userEvent.type(first_name, 'test firstname');
         userEvent.type(last_name, 'test lastname');
-        userEvent.type(date_of_birth, '2000-12-12');
-        userEvent.type(phone, '+49123456789012');
+        fireEvent.change(date_of_birth, { target: { value: '2000-12-12' } });
```

Comment:
is there any reason to use fireEvent instead of userEvent?

Reply:
The test case fails with that implementation